invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm_requests/menuTabInit.js

```diff
@@ -0,0 +1,22 @@
+import $ from "jquery";
+
+const tabElementSelector = "#request-community-submission-tab .item";
+const tabElement = $(tabElementSelector);
```

Comment:
Super minor, no change needed, for the future:
```suggestion
const $tabElement = $(tabElementSelector);
```
so that you can immediately see that it is a `jQuery` obj